.github/workflows/self-hosted-gpu-test.yml

```diff
@@ -22,96 +21,93 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.AWS_REGION }}
-      - name: Start EC2 runner
+      - name: Try to start EC2 runner
         id: start-ec2-runner
         uses: machulav/ec2-github-runner@main
         with:
           mode: start
           github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
-          ec2-image-id: ami-034abd52dd90b8023
-          ec2-instance-type: p2.xlarge
-          subnet-id: subnet-0e82552b8c708a999
-          security-group-id: sg-0589e74ec03965add
+          ec2-image-id: ami-04d16a12bbc76ff0b
+          ec2-instance-type: g4dn.xlarge
+          subnet-id: subnet-0dee8543e12afe0cd # us-east-1a
+          security-group-id: sg-0f9809618550edb98
           # iam-role-name: self-hosted-runner # optional, requires additional permissions
           aws-resource-tags: > # optional, requires additional permissions
             [
               {"Key": "Name", "Value": "ec2-github-runner"},
               {"Key": "GitHubRepository", "Value": "${{ github.repository }}"}
             ]
+
   do-the-job:
     name: Do the job on the runner
     needs: start-runner # required to start the main job when the runner is ready
     runs-on: ${{ needs.start-runner.outputs.label }} # run the job on the newly created runner
+    timeout-minutes: 1200 # 20 hrs
     env:
       TEST_MODE: GPU
-      OPENMM: ${{ matrix.cfg.openmm }}
+      OPENMM: 8.0
```

Comment:
We will need this for when we merge in https://github.com/choderalab/perses/pull/1186